nrt/data/SMIB/SMIB_BasicTestCases/SMIB_1_StepPm_IIDM/SMIB.dyd

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
2022 + description of dynawo (same everywhere)

Reply:
done